src/conq/data_recorder.py

```diff
@@ -85,7 +84,7 @@ def add_instruction(self, text: str):
         self.latest_instruction = text
         self.latest_instruction_time = time.time()
 
-    def save_episode_worker(self, robot_state_client, done: Event, mode: str):
+    def save_episode_worker(self, clients, done: Event, mode: str, save_interval):
```

Comment:
This was a smart addition. No magic hard-coded numbers! :)

Reply:
Could you rename it to `episode_save_interval` or something similar? That way the name of the variable is more indicative of its role. What's your opinion of this?

Reply:
That is a good suggestion! Done in 59c09685c33c0b7ec69632b2e5e3496dd273b061